sirepo/events.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+u"""Sirepo event registration and emittance.
+
+This module handles registering callbacks for events and calling the callbacks
+when the event occurs.
+
+For example, when a user logs out the AUTH_LOGOUT event is emitted. Other
+areas of the code can register a callback for this event (ex to clear jupyterhub
+cookies so the user is logged out of jupyterhub too).
```

Comment:
Each event should be documented here:

* 'auth_logout'  is emitted when a user logs out, before the cookie is cleared. `kwargs` contains `uid`

Reply:
21737357